### ISSUE TITLE
Breaks should be added.

### ISSUE THREAD

**Issue by JoepVanlier** [JoepVanlier/Hackey-Patterns#24]
Break functionality should be added.

**Comment by michaelsjackson** [comment]
If you add this, sequencer would be almost complete, from usefulness point of view at least. Rest would be 'only' addons.  :)   But break is very essential, imo.

**Comment by JoepVanlier** [comment]
Started implementing these. The user can make OFF markers, but Hackey Trackey is not yet able to deal with these tiny MIDI items. I am aware of the issues with that (causes an error when HT is opened on an OFF/break item).

**Comment by michaelsjackson** [comment]
Yes, I am sure it works there, here I have the gui not drawn bug, reaper 32bit windows in wine in ubuntu 64bit.
